code/platformio.ini

```diff
@@ -2802,16 +2802,18 @@ upload_port = ${common.upload_port}
 upload_flags = ${common.upload_flags}
 extra_scripts = ${common.extra_scripts}
 
-[env:gosund-sp1-v23]
+[env:gosund-ws1]
```

Comment:
Please do not delete existing gosund-sp1-v23 env :)

Reply:
Oops, fixed. Thanks!